src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarm.java

```diff
@@ -487,8 +490,45 @@ public ListSuitesResult listSuites(String runArn) {
 
         return api.listSuites(request);
     }
-
-
+    
+    /*
+     * Check if account has unmetered devices based on upload type
+     */
+    public void checkAccountUnmeteredStatus(String appArtifact) throws AWSDeviceFarmException {
```

Comment:
Is there a reason this returns void and throws an exception instead of returning a boolean?

You are "checking" if the account supports unmetered status, this doesn't seem like an "exceptional" case.